concent_api/core/payments/backends/sci_backend.py

```diff
@@ -100,27 +101,40 @@ def get_list_of_payments(
 def make_settlement_payment(
     requestor_eth_address: str,
     provider_eth_address: str,
-    value: int,
+    value: List[int],
+    subtask_ids: List[str],
     closure_time: int,
+    v: List[int],
+    r: List[bytes],
+    s: List[bytes],
+    reimburse_amount: int,
 ) -> str:
     """
-    Makes forced transaction from requestor's deposit to provider's account on amount 'value'.
-    If there is less then 'value' on requestor's deposit, Concent transfers as much as possible.
+    Makes forced transaction from requestor's deposit to provider's account on amount 'reimburse_amount'.
+    If there is less then 'reimburse_amount' on requestor's deposit, Concent transfers as much as possible.
     """
     assert isinstance(requestor_eth_address, str) and len(requestor_eth_address) == ETHEREUM_ADDRESS_LENGTH
     assert isinstance(provider_eth_address, str) and len(provider_eth_address) == ETHEREUM_ADDRESS_LENGTH
     assert isinstance(closure_time, int) and closure_time >= 0
 
-    validate_value_is_int_convertible_and_positive(value)
+    validate_value_is_int_convertible_and_positive(reimburse_amount)
+
+    for index, subtask_id in enumerate(subtask_ids):
```

Comment:
personally I would just use list comprehension in `force_payment`'s call:
```python
    return PaymentInterface().force_payment(  # type: ignore  # pylint: disable=no-member
        requestor_address=Web3.toChecksumAddress(requestor_eth_address),
        requestor_address=Web3.toChecksumAddress(requestor_eth_address),
        provider_address=Web3.toChecksumAddress(provider_eth_address),
        provider_address=Web3.toChecksumAddress(provider_eth_address),
        value=int(value),
        value=value,
        subtask_id=[_hexencode_uuid(subtask_id) for subtask_id in subtask_ids]
        v=v,
        r=r,
        s=s,
        reimburse_amount=reimburse_amount,
        closure_time=closure_time,
        closure_time=closure_time,
```
but it's optional - your call! ;)